src/org/traccar/reports/Summary.java

```diff
@@ -60,7 +63,14 @@ private static SummaryReport calculateSummaryResult(long deviceId, Date from, Da
                 speedSum += position.getSpeed();
                 result.setMaxSpeed(position.getSpeed());
             }
-            result.setDistance(ReportUtils.calculateDistance(firstPosition, previousPosition));
+            boolean ignoreOdometer = false;
+            if (device.getAttributes().containsKey(ReportUtils.IGNORE_ODOMETER)) {
```

Comment:
Maybe we can have two functions:
1. `lookupServerAttribute` - device -> group -> server
2. `lookupConfigAttribute` - device -> group -> config

In both cases group lookup will be configurable.